fidelisnetwork.json

```diff
@@ -0,0 +1,1073 @@
+                        0
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.virusTotalDetections",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.vlanId",
+                    "example_values": [
+                        -1
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.workflowAggAlertId",
+                    "example_values": [
+                        14
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "example_values": [
+                        1
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "example_values": [
+                        1
+                    ],
+                    "data_type": "numeric"
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "delete alert",
+            "description": "Delete alerts from Fidelis Network",
+            "type": "generic",
+            "verbose": "This action will always succeed regardless of the input. This action will fail if <b>alert_id</b> is not specified.",
+            "identifier": "delete_alert",
+            "read_only": false,
+            "parameters": {
+                "alert_id": {
+                    "description": "Alert IDs (comma-separated)",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "alert id"
+                    ],
+                    "order": 0
+                }
+            },
+            "render": {
+                "type": "table",
+                "width": 12,
+                "height": 5,
+                "title": "Delete Alerts"
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.alert_id",
+                    "data_type": "string",
```

Comment:
Can we include some example values here?

Reply:
Yes, I added example value.